src/System.AppContext/ref/System.AppContext.csproj

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
```

Comment:
I added CheckForOverflowUnderflow to 11 projects before this discussion started and all outerloop test were clean (though of course that could have been coincidence). I could, of course, simply try adding it to the main dir.projs file and see what happens.